content-for/head-footer.html

```diff
@@ -1,5 +1,19 @@
 <style>
   .axe-violation {
-    border: 2px solid red !important;
+    background-image: repeating-linear-gradient(
+      45deg,
+      transparent,
+      transparent 0.4em,
+      hsla(69, 84%, 83%, 1.00) 0.4em,
+      hsla(69, 84%, 83%, 1.00) 0.8em,
+      hsla(166, 44%, 65%, 1.00) 0.8em,
+      hsla(166, 44%, 65%, 1.00) 1.2em,
+      hsla(214, 96%, 45%, 1.00) 1.2em,
+      hsla(214, 96%, 45%, 1.00) 1.6em
+    );
```

Comment:
Ah, I wanted to bring this up as an open question as well. 

Part of me feels like it would be good to leave off `!important` here. The low specificity isn't too concerning so long as no one defines their own `.axe-violation` rules (i.e: we're not just trying to target `div` or something) -- and if they do, that may well be a valid use case that we shouldn't get in the way of. 

But then I can see the other side: `.axe-violation` is our special sauce -- and someone using `ember-a11y-testing` shouldn't try to use it for their own good, as it would only add ambiguity to their markup.